Hmac.js

```diff
@@ -66,12 +105,45 @@ export class Hmac {
    * @param {Uint8Array} options.data - The data to sign as a Uint8Array.
    * @param {string} options.signature - The base64url-encoded signature
    *   to verify.
+   * @param {boolean} options.useCache - Enable the use of a cache.
    *
    * @returns {Promise<boolean>} `true` if verified, `false` if not.
    */
-  async verify({data, signature}) {
+  async verify({data, signature, useCache = true}) {
+    const cacheKey = `verify-${base64url.encode(data)}`;
+    if(useCache) {
+      const verified = this.cache.get(cacheKey);
```

Comment:
It might be strange to use the same cache for signing and verifying ... but we'll find that out later if it is and change it then! This is fine for now, just making a note of it.